### PR TITLE
Increase wait for hellohttp load balancer service to come up

### PR DESCRIPTION
**Pull request by smukherj1**

### CODE REVIEW

File: examples/hellohttp/e2e-test.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ check_msg() {
    # Wait some more for after IP address allocation for the service to come
    # alive
    echo "Got IP Adress! Sleeping 30s more for service to come alive..."
```
Comment (erain) [inline examples/hellohttp/e2e-test.sh]: change the comment in echo as well?

Comment (smukherj1) [reply examples/hellohttp/e2e-test.sh]: Done locally. Will push once/if CI passes

**Comment by smukherj1** [comment]
Well seems like waiting doesn't help

**Comment by smukherj1** [comment]
Ok I think I see the issue. We are fetching the cluster IP in the hellohttp e2e test whereas we should be fetching the external endpoint IP

**Comment by smukherj1** [comment]
Looks like there's an issue with the hellohttp python server image. I tried to run it locally and I get:
```
suvanjan@suvanjan:~/repos/gob/toolchain/kokoro/generator$ docker run -p 8080:8080 us.gcr.io/rules_k8s/prow/hello-http-image@sha256:13f60aa810e954c5039fdb24313dfbcdceb9883b72c8c2677d86957a2313021f
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
 * Restarting with stat
Traceback (most recent call last):
  File "/app/examples/hellohttp/py/server.binary.runfiles/io_bazel_rules_k8s/examples/hellohttp/py/server.py", line 25, in <module>
    app.run(host='0.0.0.0', port=8080, debug=True)
  File "/app/examples/hellohttp/py/server.binary.runfiles/pypi__Flask_0_12_4/flask/app.py", line 841, in run
    run_simple(host, port, self, **options)
  File "/app/examples/hellohttp/py/server.binary.runfiles/pypi__Werkzeug_0_15_1/werkzeug/serving.py", line 988, in run_simple
    run_with_reloader(inner, extra_files, reloader_interval, reloader_type)
  File "/app/examples/hellohttp/py/server.binary.runfiles/pypi__Werkzeug_0_15_1/werkzeug/_reloader.py", line 332, in run_with_reloader
    sys.exit(reloader.restart_with_reloader())
  File "/app/examples/hellohttp/py/server.binary.runfiles/pypi__Werkzeug_0_15_1/werkzeug/_reloader.py", line 176, in restart_with_reloader
    exit_code = subprocess.call(args, env=new_environ, close_fds=False)
  File "/usr/lib/python2.7/subprocess.py", line 168, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 8] Exec format error
```

**Comment by chrislovecnm** [comment]
@smukherj1 that error is from an osx binary or other os binary running inside a Linux container. Exec format == you did not compile a Linux binary

**Comment by smukherj1** [comment]
@chrislovecnm it's basically this python [file](https://github.com/bazelbuild/rules_k8s/blob/master/examples/hellohttp/py/server.py).

Python file is not a compiled binary so I don't think it's a target platform issue. A stackoverflow search suggests a shebang at the beginning might help which is what I'm trying. The python file is running a flask server which reloads itself. The reloading itself is failing

**Comment by chrislovecnm** [comment]
Interesting, the only time I have seen it is with a wrong binary format.

TIL

**Comment by smukherj1** [comment]
Wow that worked O__O

**Pull request merged**